module-3/web-app/src/components/Pages/Projects/Projects.module.scss

```diff
@@ -1,15 +1,39 @@
 .container {
     display: flex;
+    align-items: start;
+    align-content: center;
     margin: 0 auto;
     min-height: 90vh;
+    
+    & > a {
+        width: 265px;
```

Comment:
Why width is hardcoded? Is it going to be compatible with mobile devices? Have you ever tested your app on smaller screens? I suggest to start doing that.

Reply:
I hope auto is fine